src/service.js

```diff
@@ -32,4 +32,5 @@ if (!fs.existsSync(processed)) {
     fs.mkdirSync(processed);
 }
```

Comment:
Could have added logs to the mkdir commands so the user knows directories were created